device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/InternalClient.java

```diff
@@ -353,6 +356,10 @@ public DeviceClientConfig getConfig()
      *	      option specifies the interval in milliseconds between calls to
      *	      the service checking for availability of new messages. The value
      *	      is expected to be of type {@code long}.
+     *	    - <b>SetReceiveInterval</b> - this option is applicable to all protocols
+     *	      in case of HTTPS protocol, this option acts the same as {@code SetMinimumPollingInterval}
+     *	      in case of MQTT and AMQP protocols, this option specifies the interval in millisecods between
+     *	      spawning up threads that would read a message off of the connection and invoking the message callback.
      *	    - <b>SetCertificatePath</b> - this option is applicable only
```

Comment:
"... between spawning a thread that dequeues a message from the SDK's queue of received messages..." Just to be clear that these messages aren't being taken directly from the connection, but from the SDK's transport layer that already received the message